packages/extension/src/libs/utils/updateAndSync/updateCoinSet.ts

```
@@ -0,0 +1,253 @@
+    }
+
+    isRunning = true;
+    try {
+      const updates = await syncCoinSetsOnce();
+
+      if (updates.length) {
+        options?.onUpdate?.(updates);
+
+        updates
+          .filter(update => update.containsMyCoins)
+          .forEach(update => options?.onMyCoinDetected?.(update));
```

Biome (2.3.13)

[error] This callback passed to forEach() iterable method should not return a value.

Either remove this return or remove the returned value.


(lint/suspicious/useIterableCallbackReturn)